client/my-sites/plans/jetpack-plans/get-purchase-url-callback.ts

```diff
@@ -149,8 +149,8 @@ export const getPurchaseURLCallback =
 			const yearlySlug = getYearlySlugFromMonthly( slug );
 			return yearlySlug ? buildCheckoutURL( siteSlug, yearlySlug, urlQueryArgs ) : undefined;
 		}
-		if ( purchase ) {
-			const relativePath = managePurchase( siteSlug, purchase.id );
+		if ( siteSlug && purchase ) {
```

Comment:
I'm wondering what happens if `siteSlug` is empty?

Reply:
The link would be broken since it is [just a concat string for the URL](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/purchases/paths.ts#L4). That's why I think there should be a site slug check.

Reply:
Reverted the siteSlug check in https://github.com/Automattic/wp-calypso/pull/80790/commits/8d04e4fbeefce05da6db1a92c5ba8f125097e9bd.

Reply:
Reverted the `siteSlug` check in https://github.com/Automattic/wp-calypso/pull/80790/commits/8d04e4fbeefce05da6db1a92c5ba8f125097e9bd. cc @kangzj